### PR TITLE
Refs #31267 - align recent jobs card with the current design

### PR DESCRIPTION
**Pull request by amirfefer**
Screen Shot 2021-06-03 at 18 43 16

### CODE REVIEW

**Review by adamruzicka — CHANGES_REQUESTED** [review]
Looks nice, but lints are failing

**Comment by amirfefer** [comment]
@adamruzicka, thanks.
Updated

**Comment by xprazak2** [comment]
Ruby test failures seem unrelated.

File: webpack/react_app/components/RecentJobsCard/constants.js [hunk 1]
```diff
@@ -1 +1,12 @@
...
+export const FINISHED_TAB = 0;
+export const RUNNING_TAB = 1;
+export const SCHEDULED_TAB = 2;
+
+export const JOB_SUCCESS_STATUS = 0;
+export const JOB_ERROR_STATUS = 1;
+
+export const JOB_BASE_URL = '/job_invocations?search=host+%3D+';
+export const JOB_API_URL =
+  '/api/job_invocations?order=start_at+DESC&search=targeted_host_id%3D$';
```
Comment (adamruzicka) [inline webpack/react_app/components/RecentJobsCard/constants.js:11]: I think the dollar at the end shouldn't be there. Or at least on my machine I see requests like `http://localhost:3000/api/job_invocations?order=start_at DESC&search=targeted_host_id=$8` and there's no host with ID `$8`

**Review by adamruzicka — CHANGES_REQUESTED** [review]
One thing I noticed.

Also could you rebase? It should resolve the ruby test failures

**Comment by amirfefer** [comment]
Thanks @adamruzicka ! fixed :)

**Comment by adamruzicka** [comment]
If there are no jobs, it simply shows nothing. Shouldn't there be something like a miniature empty state or something? I'm not instisting on it, just curious.

image

**Comment by amirfefer** [comment]
Thanks @adamruzicka, done
Screen Shot 2021-07-26 at 2 20 08

**Review by adamruzicka — APPROVED** [review]
Looks nice, ACK

**Comment by adamruzicka** [comment]
Thank you @amirfefer !

**Pull request merged**